browser/js/views/lecture/lectureModal.html

```diff
@@ -3,7 +3,17 @@
     <h1>Create Lecture</h1>
   </div>
   <div class="modal-body">
-    <input class="mainQuestion form-control" type="text" ng-model='lectureName' ng-keyup="$event.keyCode == 13 ? submitLecture() : null" placeholder='Lecture Title'>
+    <input class="mainQuestion form-control" type="text" ng-model='lectureName' placeholder='Lecture Title'>
+    <select id='lectureEmail' ng-model='lectureEmail'>
```

Comment:
Just noticed "Inputs and selects have `width: 100%;` applied by default in Bootstrap" [here](http://getbootstrap.com/css/#forms-inline). So maybe you don't even need to apply the styling.